demos/TicTacToe/Classes/MainMenuScene.cpp

```diff
@@ -16,6 +16,8 @@
 
 #include <regex>
 
+#include "cocos2d.h"
+#include "util.h"
 #include "TicTacToeScene.h"
```

Comment:
should probably have consistency of capitalization in the file names.

Reply:
The util.h will match the updated naming convention to follow google's style guide